src/pages/api/submit.ts

```diff
@@ -9,6 +9,15 @@ export default async function handler(
     res.status(405).end(`Method ${req.method} Not Allowed`)
   }
   const data = req.body
+  const requiredParameters = [
```

Comment:
I was checking this on frontend too, the submit button is keeps deactivated if some of the requiredParameters are missing